src/Type/Nette/CachingFallbacksDynamicReturnTypeExtension.php

```diff
@@ -0,0 +1,55 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Type\Nette;
+
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\MethodReflection;
+use PHPStan\Reflection\ParametersAcceptor;
+use PHPStan\Reflection\ParametersAcceptorSelector;
+use PHPStan\Type\DynamicMethodReturnTypeExtension;
+use PHPStan\Type\Type;
+
+final class CachingFallbacksDynamicReturnTypeExtension implements DynamicMethodReturnTypeExtension
+{
+
+	/** @var array<string, int> */
+	private $fallbackMethods = [
+		'load' => 1,
+		'call' => 0,
+		'wrap' => 0,
+	];
+
+	public function getClass(): string
+	{
+		return 'Nette\Caching\Cache';
+	}
+
+	public function isMethodSupported(MethodReflection $methodReflection): bool
+	{
+		$methodName = $methodReflection->getName();
+
+		return array_key_exists($methodName, $this->fallbackMethods) || $methodName === 'save';
```

Comment:
What's different about `save`? This must surely crash on `$this->fallbackMethods[$methodReflection->getName()]` in `getTypeFromMethodCall`

Reply:
oups, this should not be there anymore :)